base/ca/src/main/java/com/netscape/ca/ExternalProcessKeyRetriever.java

```diff
@@ -102,7 +102,7 @@ private Result parseResult(InputStream in) throws IOException {
         String result = new String(in.readAllBytes());
         logger.debug("ExternalProcessKeyRetriever: Result:\n" + result);
 
-        JsonNode root = new JSONObject(result).getRootNode();
+        JsonNode root = new JSONObject(result).getJsonNode();
```

Comment:
Yes, this was a mistake, to access read-in data you need `getJsonNode()`. Maybe the var shouldn't be called `root`, `resultNode` maybe? I'm not too concerned about that so I'm approving.